.github/renovate.json

```diff
@@ -1,79 +1,124 @@
 {
-  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": [
-    "config:recommended"
-  ],
-  "branchPrefix": "renovate/",
-  "baseBranches": ["master", "rhoai-2.16"],
-  "ignoreTests": true,
-  "automergeType": "pr",
-  "automerge": true,
-  "enabledManagers": ["dockerfile", "tekton", "rpm"],
-  "dockerfile": {
-    "schedule": ["at any time"],
-    "enabled": true,
-    "additionalBranchPrefix": "{{baseBranch}}/",
-    "branchPrefix": "konflux/mintmaker/",
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "extends": ["config:recommended"],
+    "branchPrefix": "renovate/",
+    "baseBranches": ["main", "rhoai-2.16", "rhoai-2.17"],
     "ignoreTests": true,
     "automergeType": "pr",
     "automerge": true,
+    "enabledManagers": ["dockerfile", "tekton", "rpm"],
     "packageRules": [
       {
-        "matchUpdateTypes": ["major", "minor", "pin", "pinDigest", "patch", "bump"],
-        "enabled": false
-      },
-      {
+        "matchManagers": ["dockerfile"],
```

Comment:
so, if we merge this, renovate will be only checking dockerfiles and nothing else? thats actually a good improvement, imo

we probably want to later also configure go for the two relevant kf components we build here